aiohttp/connector.py

```diff
@@ -412,14 +420,19 @@ def _close(self) -> None:
             for data in self._conns.values():
                 for proto, t0 in data:
                     proto.close()
+                    waiters.append(proto.closed)
 
             for proto in self._acquired:
                 proto.close()
+                waiters.append(proto.closed)
 
+            # TODO (A.Yushovskiy, 24-May-2019) collect transp. closing futures
```

Comment:
incomplete task ?

Reply:
work with transports here, especially `self._cleanup_closed_transports`, seem to be a workaround of the non-closed transports for SSL only:
https://github.com/aio-libs/aiohttp/blob/dcce5ea779a580c05c2b0f0555ea5cccdb4aea29/aiohttp/connector.py#L345-L348

Also, transports in `_cleanup_closed_transports` are of type `asyncio.Transport` so we can not modify them to save closing future same way as we did in this PR with `ResponseHandler`, thus I don't see a way to get the awaitable result of `transport.abort()`.

Good news is that most likely these calls are redundant since we close all protocols (i.e., save them to the list of futures: `waiters.append(proto.closed)`)